docs: Add a description of the customize format (#115)
masutaka/github-nippou: README.md

* [Dockerize](https://github.com/masutaka/github-nippou/pull/66) by @[masutaka](https://github.com/masutaka) **merged!**
```

## Optional: Customize Output Format

Customize the list output format as needed. Configurations are stored in a Gist.   
Running `github-nippou init` creates your Gist and adds its ID to `github-nippou.settings-gist-id`.

> View the default configuration [here](https://github.com/masutaka/github-nippou/blob/master/config/settings.yml).

Reviewer: 正確には "master" には "v4.2.4" などが入るので、相対パスにしました。 32d70333ed51fce008b582078f1e54b8c5902e69

https://github.com/masutaka/github-nippou/blob/v4.2.4/lib/settings.go#L168-L170

Author: おお大事ですね、よさそうです 👍🏻

### Available Properties

#### `format.subject`

| Property | Type | Description |
| --- | --- | --- |
| `subject` | `string` | Represents the repository name. |

#### `format.line`

| Property | Type | Description |
| --- | --- | --- |

> | `user` | `string` | Displays the username who interacted with the issue or pull request. |

Author: github-nippouの場合アクションをするのは常に自分であり、ここはissueやPRのauthorかも？

```suggestion
| `user` | `string` | Displays the username of author of the issue or pull request. |
```
